src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -2099,6 +2099,9 @@
   <data name="Encoding" xml:space="preserve">
     <value>Encoding</value>
   </data>
+  <data name="EncodingDetected" xml:space="preserve">
+    <value> (detected)</value>
```

Comment:
```suggestion
    <value>{0} (detected)</value>
```
It's best to include the variable in the resource as the order sometimes changes based on the translation.